plugins/modules/openssl_pkcs12.py

```diff
@@ -27,10 +27,19 @@
         choices: [ export, parse ]
     other_certificates:
         description:
-            - List of other certificates to include. Pre 2.8 this parameter was called C(ca_certificates)
+            - List of other certificates to include. Pre Ansible 2.8 this parameter was called C(ca_certificates)
+            - Assumes there is one PEM-encoded certificate per file. If a file contains multiple PEM certificates,
+              set I(other_certificates_parse_all) to C(true).
         type: list
         elements: path
         aliases: [ ca_certificates ]
+    other_certificates_parse_all:
+        description:
+            - If set to C(true), assumes that the files mentioned in I(other_certificates) can contain more than one
+              certificate per file (or even none per file).
+        type: bool
+        default: false
```

Comment:
```suggestion
```
;)

Reply:
The option is defined as `other_certificates_parse_all=dict(type='bool', default=False),` in the argspec, so removing the default here would be wrong, wouldn't it?

Reply:
Oh, i didn't sleep well because of SpaceX;) I mixed it up with `required: no`, sorry:) Irrelevant